person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name, age, parent_permission: true)
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_library?
```

Comment:
- [ ] The name of this method should be `can_use_services?` as clearly stated in the requirements.